plugins/view-transitions/includes/settings.php

```diff
@@ -64,6 +65,7 @@ function plvt_get_setting_default(): array {
 		'post_title_selector'          => '.wp-block-post-title, .entry-title',
 		'post_thumbnail_selector'      => '.wp-post-image',
 		'post_content_selector'        => '.wp-block-post-content, .entry-content',
+		'admin_transition_animation'   => false,
```

Comment:
Since this is a boolean for whether to use view transitions in WP Admin, I think we should give this a different name. Right now, it could be easily mistaken for an option for _which_ animation to use for WP Admin, which it isn't.

How about `enable_admin_transitions`?